spec/features/form/saving_data_spec.rb

```diff
@@ -88,11 +88,11 @@
     # Something about our styling makes the selenium webdriver think the actual radio buttons are not visible so we pass false here
```

Comment:
Looks like this comment can probably go now?

Reply:
done